doc/kubernetes/modules/ROOT/pages/running/keycloak-with-external-infinispan-deployment.adoc

```diff
@@ -0,0 +1,60 @@
+= Keycloak with external Infinispan Deployment
+:description: This describes configuration tweaks and changes on how to run Keycloak with an external Infinispan Deployment under load.
+
+{description}
+
+== Audience
+
+This guide describes advanced Infinispan configurations for Keycloak on Kubernetes which are load tested and will recover from single Pod failures.
+
+While the Helm charts in the Keycloak Benchmark project mix different aspects of production style deployments with instrumentation and monitoring, this documentation focuses on a minimal deployment with optional add-ons which admins can opt in for their own deployments.
+
+See xref:running/index.adoc[] for additional guides.
+
+== Prerequisites
+
+* OpenShift or Kubernetes cluster running.
+* Understanding of the xref:running/keycloak-deployment.adoc[Basic Keycloak deployment].
+* Understanding of a https://infinispan.org/docs/helm-chart/main/helm-chart.html[Infinispan deployment using Helm charts].
```

Comment:
We are not using the Infinispan helm charts but the Infinispan operator: https://infinispan.org/docs/infinispan-operator/main/operator.html

Reply:
Good point, I will update the resource and change the language to indicate that.